code/datums/diseases/transformation.dm

```diff
@@ -287,6 +287,22 @@
 	new_form = /mob/living/simple_animal/hostile/morph
 	infectable_biotypes = MOB_ORGANIC|MOB_INORGANIC|MOB_UNDEAD //magic!
 
+/datum/disease/transformation/morph
+	name = "Carcinization"
+	cure_text = /datum/reagent/cellulose
+	cures = list(/datum/reagent/cellulose) // opposite of crab = barc = bark = cellulose
+	agent = "Carcinization"
+	desc = "Nature's next attempt to evolve you into a crab."
+	stage_prob = 20
+	severity = DISEASE_SEVERITY_BIOHAZARD
+	visibility_flags = 0
+	stage1	= list("Your hands itch.")
+	stage2	= list("Your skin feels crusty.")
+	stage3	= list(span_danger("Your hands are forming into claws."), span_danger("Your limbs begin to become crablike."))
+	stage4	= list(span_danger("You're raveing."))
```

Comment:
this would be spelled without the "e" 👀

Reply:
No its spelt raveing intentionally

as in rave

as in https://www.youtube.com/watch?v=LDU_Txk06tM